plugins/mcreator-localization/help/fr_FR/biome/generate_ruined_portals.md

```diff
@@ -0,0 +1 @@
+Sélectionner ce paramètre pour générer des ruines de portail du Nether.
```

Comment:
Instead of using a "groupe nominal" as a complement, you just use a "group prépositionnel", so this changes absolutely nothing (it's not even a better way to say this).